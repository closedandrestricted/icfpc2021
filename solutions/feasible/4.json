{"vertices":[[13,31],[21,20],[5,27],[18,27],[15,24],[18,16],[31,18],[7,10],[5,6],[8,13],[13,14],[5,30],[8,20],[7,17],[37,39],[47,26],[38,31],[13,22],[11,15],[22,28],[20,21],[16,19],[38,17],[36,18],[19,11],[14,9],[49,25],[25,16],[28,20],[15,16],[10,15],[40,35],[10,14],[7,11],[33,59],[10,7],[5,5],[38,16],[28,35],[26,39],[19,13],[32,61],[22,29]]}