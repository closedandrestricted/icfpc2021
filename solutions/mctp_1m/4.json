{"vertices":[[30,86],[32,72],[42,64],[32,80],[29,66],[60,60],[34,66],[26,51],[31,49],[70,64],[65,66],[43,54],[75,59],[68,59],[42,16],[40,30],[33,36],[80,60],[73,58],[59,46],[84,55],[80,57],[68,30],[36,17],[92,50],[92,45],[34,32],[25,20],[30,22],[92,43],[89,39],[51,30],[91,38],[85,36],[19,13],[86,34],[84,29],[62,18],[46,17],[51,17],[79,26],[19,13],[60,22]]}