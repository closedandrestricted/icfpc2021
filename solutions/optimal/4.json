{"vertices":[[5,5],[35,15],[65,15],[95,5],[95,50],[70,70],[50,95],[30,90],[30,70],[68,80],[69,85],[57,83],[66,87],[62,84],[39,71],[29,62],[31,52],[70,90],[64,79],[70,72],[69,83],[65,86],[52,75],[40,52],[61,78],[59,83],[25,59],[51,53],[55,50],[67,76],[66,82],[55,57],[63,78],[64,77],[5,50],[58,74],[57,79],[73,51],[33,58],[38,60],[63,66],[47,38],[46,66]]}