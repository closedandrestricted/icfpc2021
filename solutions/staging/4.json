{"vertices":[[6,27],[9,11],[17,39],[8,33],[14,15],[25,53],[5,49],[26,23],[31,25],[30,45],[34,47],[30,70],[35,40],[37,53],[38,35],[54,33],[46,26],[40,42],[39,49],[47,76],[47,43],[46,48],[43,92],[50,18],[57,47],[56,52],[35,15],[62,18],[62,23],[64,47],[53,45],[62,88],[69,48],[58,45],[95,5],[66,54],[62,51],[70,70],[79,28],[78,33],[75,52],[95,50],[69,37]]}