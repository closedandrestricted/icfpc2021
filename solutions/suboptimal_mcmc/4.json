{"vertices":[[76,57],[63,48],[70,70],[83,59],[59,42],[55,81],[95,50],[62,56],[64,51],[46,77],[50,75],[69,90],[41,73],[44,72],[95,5],[81,13],[74,21],[36,73],[39,72],[50,94],[30,70],[32,75],[31,90],[65,15],[37,62],[39,58],[80,19],[54,20],[49,22],[41,56],[44,62],[42,74],[44,52],[44,57],[5,50],[40,46],[41,51],[43,55],[23,27],[28,25],[38,36],[5,5],[35,17]]}