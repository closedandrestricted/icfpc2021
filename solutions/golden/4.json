{"vertices":[[21,38],[33,27],[12,37],[16,34],[38,31],[31,33],[15,50],[26,41],[30,38],[28,23],[32,21],[15,35],[33,18],[27,16],[65,91],[51,87],[41,89],[38,18],[29,21],[34,33],[31,20],[34,16],[48,44],[40,78],[29,29],[28,24],[47,93],[38,68],[40,63],[34,24],[21,27],[33,52],[34,29],[26,25],[65,22],[30,35],[26,32],[51,49],[60,51],[57,55],[40,34],[86,47],[48,52]]}